polls/models.py

```diff
@@ -17,8 +16,16 @@ def choices(self):
 
 
 class Choice(Page):
-    votes = models.IntegerField(default=0)
```

Comment:
Please leave this field as as, otherwise you'll have to do a data migration for the data in production.